script/ArcPay.s.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/ArcPay.sol";
+
+contract ArcPayScript is Script {
+    function setUp() public {}
+
+    function run() public {
+        uint deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+        vm.startBroadcast(deployerPrivateKey);
+        console2.log(
+            // Owner is set to Anvil's first address.
+            address(new ArcPay(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266))
+        );
+        vm.stopBroadcast();
```

Comment:
I can't quite understand this from the forge docs: do we actually send a transaction with this code? The docs say that `startBroadcast` only "creates transactions that can later be signed and sent onchain" rather than actually signing and sending them.

Reply:
this is how you run a deployment script `forge script script/ArcPay.s.sol:ArcPayScript --sig "run()" --rpc-url $LOCAL_RPC --broadcast`.

That `--broadcast` flag is how you actually broadcast a signed transaction. Without this flag, it just simulates the transaction locally.